lib/tasks/add_freak_with_a_role.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :add_freak_with_a_role do
+  desc 'Add a freak with a role'
+  task add_freak_in_db: :environment do
```

Comment:
This does not properly describe what the method does. Make it clear, otherwise it will induce other devs in error.